Backend/server/src/main/java/com/uaic/server/SpringConfig.java

```diff
@@ -0,0 +1,24 @@
+package com.uaic.server;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.Customizer;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration
+public class SpringConfig {
+    
```

Comment:
redenumeste-l AuthConfig sau cv de genu si creeaza un folder pt security sa-l pui acolo:

Reference:
https://github.com/stefan1anuby/Employee-Management-And-Payroll-System/blob/main/Backend/server/src/main/java/com/uaic/server/README.md